src/twisted/names/test/test_dns.py

```diff
@@ -1971,6 +2019,32 @@ def test_srv(self):
             dns.Record_SRV(10, 20, 30, b'example.com', 400))
 
 
+    def test_sshfp(self):
+        """
+        Two L{dns.Record_SSHFP} instances compare equal if and only if
+        they have the same key type, fingerprint type, fingerprint, and ttl.
+        """
+        # Vary the key type
```

Comment:
I know that existing code is using the same convention.
I am ok to keep multiple assertions into this single test, but please add a full stop at the end of the comment to make it clear that the comment is done and that it was not accidentally left unfinished.